### PR TITLE
[AAE-7035] Being able to load a dynamic file location

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/cli/scripts/resources.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@
 export const ACTIVITI_CLOUD_APPS: any = {
     SUB_PROCESS_APP: {
         name: 'subprocessapp',
-        file_location: 'https://github.com/Alfresco/alfresco-ng2-components/blob/develop/e2e/resources/activiti7/subprocessapp.zip?raw=true',
+        file_location: (TAG = 'develop') => `https://github.com/Alfresco/alfresco-ng2-components/blob/${TAG}/e2e/resources/activiti7/subprocessapp.zip?raw=true`,
```
Comment (popovicsandras) [inline lib/cli/scripts/resources.ts:22]: Previously we were using it as a normal property, now as a function. How it doesn't cause any problem with other parts of the app?

**Review by popovicsandras — CHANGES_REQUESTED** [review]

File: lib/cli/scripts/resources.ts (hunk 1 above)
Comment (mauriziovitale) [reply lib/cli/scripts/resources.ts:22]: this file is only used by init-aae-env command

Comment (popovicsandras) [reply lib/cli/scripts/resources.ts:22]: in this case: 🎖️

Comment (mauriziovitale) [reply lib/cli/scripts/resources.ts:22]: what a pleasure!

**Review by arditdomi — APPROVED** [review]

**Review by andrea-ligios — APPROVED** [review]
LGTM!

**Pull request merged**